Pinta.Core/Effects/BaseEffect.cs

```diff
@@ -37,6 +37,12 @@ namespace Pinta.Core;
 [Mono.Addins.TypeExtensionPoint]
 public abstract class BaseEffect
 {
+	/// <summary>
+	/// If the ROIs are subdivided (in rows, for example) and the effect is applied to each subdivision
+	/// separately, is the end result the same as applying it to the original ROIs?
+	/// </summary>
+	public abstract bool IsTileable { get; }
```

Comment:
What do you think about defaulting this to true to reduce boilerplate, since this is the case for all existing effects?

Reply:
@cameronwhite I made the property `abstract` on purpose because I figured tileability should be a concious choice. What do you think?

Reply:
I don't have a strong opinion, so I'm fine to keep it as-is since you've already made the changes anyway